tests/Various.ipf

```diff
@@ -11,6 +11,7 @@ static Constant MMD_COMBO_COUNT = 32
 Function TriggerWarningToHistory()
 	DoWindow/K HistoryCarbonCopy
 	NewNotebook/V=0/F=0 /N=HistoryCarbonCopy
+	INFO("This warning is intended to fail. The next testcase will search for it.")
```

Comment:
It looks like the next testcase is not searching for it.
I think there is something like
```
	Notebook HistoryCarbonCopy, findText={"This warning is intended to fail. The next testcase will search for it.", 2}
	CHECK_EQUAL_VAR(1, V_flag)
```
missing in the CheckForWarningInHistory test case.